pkg/products/rhssouser/reconciler.go

```diff
@@ -61,6 +61,7 @@ const (
 	masterRealmClientName       = "master-realm"
 	firstBrokerLoginFlowAlias   = "first broker login"
 	reviewProfileExecutionAlias = "review profile config"
+	adminCredentialSecretName   = "credential-rhssouser"
```

Comment:
This needs to be dynamic based on `credential-` + keycloakName var from above

Reply:
Good catch ! Will make it combination based on the `keyCloakName` var